### DISCUSSION TITLE
25 Wallet Words

### DISCUSSION THREAD

**Discussion by palm2k** [Chia-Network/chia-blockchain#3921] [Support]
Hi there,

I just noticed something strange with my wallet words.
 
I have 25 words and it starts with "11" instead of "1". After that it goes from "1" to "24".

![Chia_Words_b_cut](https://user-images.githubusercontent.com/83760530/117283601-3b511980-ae66-11eb-8838-bed58fd57ed9.jpg) 

The first "11" and second "11" word is different. I did not find anything about this issue on the internet. What does this now mean for me and is this a problem now? Is this a known issue? To loose the progress so far would be devastating. Does someone know something about this? I hope you can help me.

**Comment by palm2k** [comment]
Well I just saw that the seed is also shown while showing the private keys. There is one word missing and I think it's clear now. But maybe it is good to know that this bug can occure while creating a wallet.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
